force-app/main/default/classes/PMT_Payment_TDTM.cls

```diff
@@ -321,4 +323,39 @@ public class PMT_Payment_TDTM extends TDTM_Runnable {
             }
         }
     }
+
+    private void handleAllocationsAndOpportunities(
+        List<npe01__OppPayment__c> newList,
+        List<npe01__OppPayment__c> oldList
+    ) {
+        List<npe01__OppPayment__c> refunds = new List<npe01__OppPayment__c>();
+        List<npe01__OppPayment__c> oldRefunds = new List<npe01__OppPayment__c>();
+        for (Integer i = 0; i < newList.size(); i++) {
+            if (newList[i].npe01__Payment_Amount__c < 0 && String.isNotBlank(newList[i].Elevate_Payment_ID__c)) {
```

Comment:
@andrewyu-salesforce since we have `String.isNoBlank(Elevate_Payment_ID__c)` here, wouldn't this mean that the only refunds included in the following logic would be Elevate refunds? Thought we wanted this logic to only run for non-Elevate refunds for this work?

Reply:
good catch!!